terraform/aws/modules/rds-aurora/main.tf

```diff
@@ -0,0 +1,101 @@
+resource "aws_rds_cluster" "this" {
+  cluster_identifier              = "${local.stack}-${var.rds_suffix}"
+  engine                          = var.db_engine_type
+  engine_version                  = var.db_engine_version
+  engine_mode                     = var.db_engine_mode
+  database_name                   = var.database_name
+  master_username                 = var.master_username
+  master_password                 = random_password.master_password.result
+  final_snapshot_identifier       = local.snapshot_name
+  skip_final_snapshot             = false
+  backup_retention_period         = var.backup_retention_period
+  preferred_backup_window         = var.backup_window
+  preferred_maintenance_window    = var.maintenance_window
+  port                            = 3306
+  storage_encrypted               = true
+  allow_major_version_upgrade     = var.allow_major_version_upgrade
+  enabled_cloudwatch_logs_exports = var.enabled_cloudwatch_logs_exports
```

Comment:
I feel we should actually hardcode the log exports as well, or at least require "audit" exports by default with an option to add the other types using variables (may have to use a terraform function to merge the list to do that). 

By security policy, we want to enforce audit logs are always turned on for services that provide them.

Reply:
^ this security requirement is new to me by the way... so there will be changes to other modules in the future to address this requirement.